servicetalk-opentracing-http/src/test/java/io/servicetalk/opentracing/http/TracingHttpRequesterFilterTest.java

```diff
@@ -189,4 +221,54 @@ private static ServerContext buildServer() throws Exception {
     private static String toStringOrNull(@Nullable CharSequence cs) {
         return cs == null ? null : cs.toString();
     }
+
+    private static final class TestTracingLoggerFilter implements StreamingHttpClientFilterFactory {
+        private final String[] logLinePrefix;
+
+        TestTracingLoggerFilter(final String[] logLinePrefix) {
+            this.logLinePrefix = requireNonNull(logLinePrefix);
```

Comment:
I guess it doesn't matter as its just used in the test but usually you should call `.clone()` on an array when it is passed via the constructor to ensure it can't be modified later on.